package.json

```diff
@@ -37,7 +37,7 @@
     "@semantic-release/release-notes-generator": "^10.0.3",
     "@types/glob": "^7.1.4",
     "dotenv": "^16.0.0",
-    "eslint": "^7.23.0",
+    "eslint": "^7.25.0",
```

Comment:
Out of curiosity - why do we only want to upgrade this two minor releases? The Caret should take care of this already. Granted it's only eslint, so not the be all and end all just wondered if it made sense to update the major version?

Reply:
Decided to just leave change as it is.